templates/default/exabgp.conf.erb

```diff
@@ -1,10 +0,0 @@
-neighbor 127.0.0.1 {
```

Comment:
This template is still required as a default: https://github.com/dnsimple/chef-exabgp/blob/5ab6b6fc03deeab874273ea99488fe890e7d569b/resources/exabgp.rb#L34

Reply:
The fact you removed it without a test breaking reveals a regression we should be covering in our test cases.